Skills.md

```diff
@@ -38,6 +39,24 @@ JSON
   ]
 }
 ```
+If the return_relevant_skills request parameter is set to true, the reponse will look like this:
+
+```
+{
+	"response": [{

```

Comment:
I would recommend this one:
http://www.freeformatter.com/json-formatter.html#ad-output
I love this since it gives you different format options. Especially, I like the one that outputs to one compact line, so you can easily put in unit tests.